pages/index.tsx

```diff
@@ -18,7 +18,8 @@ const Index = () => {
         <div className="m-auto w-[300px] md:w-[400px] lg:w-[500px] bg-gray-lighter">
           <div className="p-8">
             <h1 className="sr-only">
-              service.canada.ca-passport-status-checker
+              Passport application status checker | Outil de vérification de
+              l&apos;état de la demande de passeport
```

Comment:
`&apos;` should be changed to `'` to be consistent with the rest of the application.

Reply:
We are actually using `&#39;` in other pages. Eslint will complains if we don't escape `'` in the html.
![image](https://user-images.githubusercontent.com/114004123/213742964-a502b5eb-16d7-469d-86f4-377d01286b19.png)

Reply:
I've updated that file with `&#39;`

Reply:
Because it's a bilingual page and not translated.